crates/forge/bin/cmd/test/mod.rs

```diff
@@ -239,6 +239,15 @@ impl TestArgs {
             project = config.project()?;
         }
 
+        // Set number of threads to use for running tests.
+        // If not specified then the number of threads determined by rayon will be used.
+        if let Some(test_threads) = config.max_test_threads {
```

Comment:
Note that this will affect any rayon operation, not just testing, which is fine.
Wonder if this should be a global option, like `-j`?

Reply:
:+1: make sense, moved it as global `--max-threads` test option ![image](https://github.com/foundry-rs/foundry/assets/38490174/54533bf1-8362-439d-ae26-3f29262328e1) with https://github.com/foundry-rs/foundry/pull/7957/commits/af92d6dfd92b21b38f1d2dfff30150d014b17b05